src/main/java/com/example/AllPermutationPrime.java

```diff
@@ -0,0 +1,62 @@
+package com.example;
+
+import java.util.ArrayList;
+
+/**
+ * Készíts egy olyan függvényt, aminek
+ * bemenő paramétere n,
+ * és válaszként kilistázza azokat a számokat 1-től n-ig,
+ * amelyeknek az összes körkörös permutációja prím.
+ */
+public class AllPermutationPrime {
+
+    private static boolean isPrime(int n) {        
+        boolean prime = true;
+        int divider = n - 1;
+        while ((divider > 1) && prime) {
+            if (n % divider == 0) {
+                prime = false;
+            }
+            else {
+                divider--;
+            }
+        }
+        return prime;
+    }
+
+    private static int[] generateCircledPermutations(int checkedNumber) {
+        String number = String.valueOf(checkedNumber);       
+        int numberOfDigits = number.length();
+        number += number;
+        int[] permutations = new int[numberOfDigits];
+        for (int i = 0; i < numberOfDigits; i++) {
+            permutations[i] = Integer.parseInt(number.substring(i, i + numberOfDigits));
+        }
+        return permutations;
+    }
+
+    public static int[] allCircledPermutationsArePrimes(int n) {
```

Comment:
You can return a `List` here.

Reply:
In that case, I eliminate the conversion from ArrayList and return it here.